recipes-core/packagegroups/packagegroup-ni-extra.bb

```diff
@@ -78,6 +78,7 @@ RDEPENDS_${PN} += "\
 	bind \
 	cifs-utils \
 	libpcap \
+	libfmi \
```

Comment:
Since this package family is required by Veristand, could you move `libfmi` to `packagegroup-ni-coreimagrepo` and put it in an `RDEPENDS_${PN}_append` section with a comment declaring that it is a Veristand requirement, and tag Marcelo Izaguirre as the engineering contact.